canonical-websites/www.ubuntu.com
# 2740: Add animation to beta nav dropdown

## Done

- Added a slight fade in and vertical movement to beta nav dropdown
- Moved navigation javascript to navigation.js file
- Fixed formatting on Download tab

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Check that the navigation dropdowns animate
- Go to a page with breadcrumbs (e.g. /cloud) and test the animation there too
- Check that the Download tab doesn't have a second resource column

## Issue / Card

Fixes #2740
